src/main/java/com/sskkilm/cashflow/enums/GlobalErrorCode.java

```diff
@@ -0,0 +1,15 @@
+package com.sskkilm.cashflow.enums;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum GlobalErrorCode implements ErrorCode{
```

Comment:
```suggestion
public enum GlobalErrorCode implements ErrorCode {
```

java coding convention 에 해당하는 내용은 확인해보고, 지켜보는 습관을 가지는 것은 어떨까요?

현업에서는 개발자들간의 style 을 어느정도로 통일하여 같은 style 의 코드를 만드는 것에 많은 노력을 가하고 있습니다.
https://naver.github.io/hackday-conventions-java/